### DISCUSSION TITLE
Welcome to runelite-profit-tracker Discussions!

### DISCUSSION THREAD

**Discussion by nofatigue** [nofatigue/runelite-profit-tracker#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gbeard2** [comment]
Hi, I noticed that you are no longer actively developing this plugin. There are some features I would like to see added. I am willing to take over development and maintain this plugin if that's okay with you. I can fork the repo, but we will need to update the Runelite integration. Or I will need access to this repo. This plugin has been very helpful. Thank you for your initial development.

**Comment by nofatigue** [comment]
Hello,
I am glad you want to continue development. I am fine with updating the
Runelite repo to use your fork. Another option - how about a transfer of
ownership with all the existing issues?

**Reply by gbeard2** [reply]
I am going to start with the reported issues first, so that would actually be ideal.

**Reply by nofatigue** [reply]
After thinking a bit, I think I feel more comfortable to use a fork. How about you make fork with a bit of changes you'd like to see? (I don't know if you started playing with plugin code already). Then we can do what is required for the RuneLite Plugin repo to point to your repo. Again, I'm happy with your will to work with the code :)
